operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -370,56 +370,58 @@ local filename = 'mimir-writes.json';
       $._config.autoscaling.ingester.enabled,
       $.row('Ingester – autoscaling')
       .addPanel(
-        $.autoScalingActualReplicas('ingester') + { title: 'Replicas (leader zone)' } +
+        local replicaTemplateQueries = [
+          |||
+            max by (name) (
```

Comment:
[nit] Why `by (name)` given we don't display the `{{ name }}` in the legend?

_Same comment applies below._

Reply:
removed in https://github.com/grafana/mimir/pull/9617/commits/d3f47266487ff63d5707affa136a3b75a6ed3d93